client/src/components/dashboard/Preferences.js

```diff
@@ -17,14 +17,14 @@ import propTypes from 'prop-types';
 import React from 'react';
 import { savePreferencesViewState } from '../../actions/views';
 import { saveUser, updateUser, updateUserPartial } from '../../actions/user';
-import skills from '../../assets/dropdowns/skills';
+import skills, { SKILLS_MAP } from '../../assets/dropdowns/skills';
 import SkillsAndInterests from './Profile/Preferences/SkillsAndInterests';
 import Social from './Profile/Preferences/Social';
 import styled from 'styled-components';
 import swearjar from '../../assets/helpers/swearjar-lite';
 import { ThickPaddedBottom } from '../../styles/style-utils';
 import UserLabel from '../dashboard/common/UserLabel';
-import Validator from 'validator';
+import __ from '../../assets/helpers/validations';
```

Comment:
@bonham000 could... but the methods on the object are descriptive to the point where it's clear what it is, no? Again, I made this choice based on my stylistic preferences. So maybe that would be annoying to other developers. But also `validator.validateMentorship` is redundant. 

As I was typing this, I just remembered another convention from "Clean Code Javascript"...

I'll rename the variable `validate` and then each of the methods can just be the things that they're validating, e.g. `validate.mentorship`. This will solve the stylistic issues that annoyed me with a long object name plus long method name, as well as the redundancy issues.